### PR TITLE
ICMSLST-704 Move import contacts to fa module

### PR DESCRIPTION
**Pull request by kevinetienne**

### CODE REVIEW

**Review by oskusalerma — APPROVED** [review]

**Review by MattHolmes123 — CHANGES_REQUESTED** [review]

File: web/domains/case/_import/fa/views.py [hunk 1]
```diff
@@ -268,11 +269,170 @@ def add_response_constabulary_email(
...
+
+    try:
+        link = process_type_link[application.process_type]
+    except KeyError:
+        raise NotImplementedError(
+            f"Unable to check know_bought_from: Unknown Firearm process_type: {application.process_type}"
+        )
+
+    # e.g. application.openindividuallicenceapplication to get access to OpenIndividualLicenceApplication
+    firearms_application: FaImportApplication = getattr(application, link)
```
[the remark this answers is not in the document]
Comment (kevinetienne) [reply web/domains/case/_import/fa/views.py]: fixed in https://github.com/uktrade/icms/pull/412/commits/c313a0bace244791d0c789585f639e34daab1d07 I decided to get the firearms application in the view and to pass it to this function.

File: web/domains/case/_import/fa/views.py [hunk 2]
```diff
@@ -9,25 +9,26 @@
 from django.utils import timezone
 from django.views.decorators.http import require_POST
 
+from web.domains.case import views as case_views
```
[the remark this answers is not in the document]
Comment (kevinetienne) [reply web/domains/case/_import/fa/views.py]: https://github.com/uktrade/icms/pull/412/commits/c313a0bace244791d0c789585f639e34daab1d07

File: web/domains/case/_import/fa/views.py [hunk 3]
```diff
@@ -268,11 +269,170 @@ def add_response_constabulary_email(
         )
 
 
+# TODO: Revisit these if we get around to ICMSLST-657
```
[the remark this answers is not in the document]
Comment (kevinetienne) [reply web/domains/case/_import/fa/views.py]: https://github.com/uktrade/icms/pull/412/commits/c313a0bace244791d0c789585f639e34daab1d07

File: web/domains/case/_import/fa/urls.py [hunk 4]
```diff
@@ -10,8 +10,28 @@
...
+                                views.list_import_contacts,
+                                name="list-import-contacts",
+                            ),
+                            path(
+                                "<entity>/create/",
+                                views.create_import_contact,
+                                name="create-import-contact",
+                            ),
+                            path(
+                                "<entity>/<int:contact_pk>/edit/",
```
[the remark this answers is not in the document]
Comment (kevinetienne) [reply web/domains/case/_import/fa/urls.py]: grouped in https://github.com/uktrade/icms/pull/412/commits/c313a0bace244791d0c789585f639e34daab1d07

**Review by MattHolmes123 — APPROVED** [review]

**Pull request merged**